flake.nix

```diff
@@ -103,6 +102,7 @@
 
         propagatedDeps =
           [ (boehmgc.override { enableLargeConfig = true; })
+            nlohmann_json
```

Comment:
Wouldn’t that also propagate it to the non-dev outputs? (Not that it’s a big dependency ofc, but still)